README.md

```diff
@@ -55,29 +55,55 @@ The criteria for picking these event names included:
-   - `childSuites`: array with all direct subsuites
-   - `tests`: array containing all tests that directly belong to the suite (but not to a child suite)
-   - `runtime`: execution time of the whole suite in milliseconds (including child suites)
-   - `status`: summarized status of the suite
-      - `failed`, if at least one test failed
-      - `skipped`, if all tests in the suite are skipped (and there is at least one skipped test)
-      - `passed`, if there is at least one passed test and all other tests are skipped or if there are no tests in the suite.
+Based on the discussion in [#79](https://github.com/js-reporters/js-reporters/issues/79), js-reporters establishes also a minimum set of properties for the emitted assertions, failed or passed:
+
+- **Assertion**: `Object` - An object which contains information of a single assertion/expectation.
+ - **passed**: `Boolean` - `true` for a passed assertion, `failed` for a failed assertion
+ - **actual**: `*` - the actual value passed to the assertion, should coincide with `expected` for passed assertions
+ - **expected**: `*` - the expected value passed to the assertion, should coincide with `actual` for passed assertions
+ - **message**: `String` - a description
+ - **stack**: `String|undefined` - represents the stack trace for a failed assertion, for a `passed` one it is `undefined`
+
+The content of the properties will vary based on the used assertion library and of course assertion libraries can extend with other properties.
+
+Based on the discussion in [#12](https://github.com/js-reporters/js-reporters/issues/12#issuecomment-120483356), there are `two basic data structures: Suites and Tests`. A test represents an atomic test/spec/`it()`. A suite contains tests and optionally other suites. This can represent both flat structures like used by QUnit as well as nested suites like used by Jasmine or Mocha. The data structures are defined as follows:
+
+- **Test**: `Object` - A test holds basic information on a single test/spec.
+ - **name**: `String` - name of the test
+ - **suiteName**: `String` - name of the suite the test belongs to
+ - **fullName**: `Array` - array of strings containing the name of the test and the names of all its suites ancestors
+ - **status**: `String|undefined` - result of the test, it will be `undefined` for the `testStart` event. Can be:
+    - `passed`, if all assertions have passed
+    - `failed`, if at least one assertion have failed
+    - `skipped`, a skipped test is disabled, i.e. it will not be executed
+ - **runtime**: `Number|undefined` - execution time in milliseconds, it will be `undefined` for the `testStart` event.
+ - **errors**: `Array|undefined` - array containing all errors, i.e failed Assertions, it will be `undefined` for the `testStart` event. Depending on the framework, this is a single exception or a list of failed assertions. It will contain at least one error for failed statuses and it will be empty for statuses other than failed.
```

Comment:
Let's drop the "Depending on the framework..." sentence. The last sentence is clear enough.